cfme/cloud/provider/__init__.py

```diff
@@ -79,8 +81,9 @@ class CloudProviderImagesView(BaseLoggedInPage):
     """
     @property
     def is_displayed(self):
+        location = VersionPicker({Version.lowest(): 'Cloud Providers', '5.11': 'Compute'}).pick()
         return (
-            self.breadcrumb.locations[0] == 'Cloud Providers' and
+            self.breadcrumb.locations[0] == location and
```

Comment:
I think you need to go further here, not just change the expected string for the first breadcrumb item.

Breadcrumbs are significantly different in 5.11/Ivanchuck, and I think the important part here is that the breadcrumb is including 'Cloud Providers'. That the breadcrumb includes 'Compute' doesn't really help validate that we're on Cloud Provider images.

Look at asserting against the last and second-to-last items in the breadcrumb, as they are what indicate more specifics about the page being viewed. Use `self.breadcrumb.locations[-1] == self.context['obj'].name` kind of assertions, where that last item may be different between the versions.

Reply:
Hi @mshriver 
- I am agree with your approach of going with last and second-to-last items in breadcrumb.  last item is same as title of the page and also same in both versions.
- Breadcrumb in 5.11: ```Compute > Clouds > Providers > Cloud Providers > azure > azure (All Images)```
- Breadcrumb in 5.10: ```Cloud > Providers > azure (All Images)```
- Also using second-to-last item which also indicates the more specifics about the page being viewed.
So I updated is_diplayes as below:
```
    def is_displayed(self):
        location = VersionPicker({Version.lowest(): 'Cloud Providers',
                                  '5.11': self.context['object'].name}).pick()
        return (
            self.breadcrumb.locations[-2] == location and
            self.entities.title.text == self.breadcrumb.locations[-1])
```